### ISSUE TITLE
Bearings

### ISSUE THREAD

**Issue by Cullen-Dubbs** [G-Levine/OpenTorque-Actuator#1]
Hi there,

I am a little confused about the bearings. I have the 16mm OD bearings you have on the parts list. I somewhat see what the design calls for with the slot in the center for the bearing to go into. But I am not able to fit the bearing in. Do I need to place the bearing in there mid print?

**Comment by G-Levine** [comment]
Yes that’s correct, the bearing goes in mid-print. Sorry, the instructions aren’t clear. I will update them to explain it better.

**Comment by Cullen-Dubbs** [comment]
No problem! I'm having alot of fun putting this together!

**Comment by createthis** [comment]
Is there a specific height at which the print should be paused? I’m printing on an Ultimaker 2+ and I’m trying to figure out how to make this happen most efficiently.
